### DISCUSSION TITLE
Config file content collisions

### DISCUSSION THREAD

**Discussion by kayavila** [goreleaser/nfpm#451] [Q&A] [answered]
Should I be able to specify a config file location that's under a directory also included as part of a package?  I'm running into issues with this.  The entirety of my contents block from the config are listed below -

```contents:
- src: /srv/zeek/
  dst: /srv/zeek/
- src: /srv/zeek/etc/networks.cfg
  dst: /srv/zeek/etc/networks.cfg
  type: config
- src: /srv/zeek/etc/node.cfg
  dst: /srv/zeek/etc/node.cfg
  type: config
- src: /srv/zeek/etc/zeekctl.cfg
  dst: /srv/zeek/etc/zeekctl.cfg
  type: config
```
And I get this error -
```[root@146d7fe3969c build]# nfpm package -p rpm -t build/
using rpm packager...
cannot add "/srv/zeek/etc/networks.cfg" because "/srv/zeek/etc/networks.cfg" is already present at the same destination (/srv/zeek/etc/networks.cfg): content collision
```

**Comment by djgilcrease** [comment, marked as answer]
No as the file already got added to the content list as a normal file type. You should add all the other directories in `/srv/zeek/` except `etc` as individual entries in the contents, then the etc directory as a type config.

**Reply by kayavila** [reply]
I see - thank you.  I was erroneously expecting it to work like fpm.  That's not ideal, especially because one of the config files I didn't include in the sample code is not inside etc/, but quite nested under other directories, but it works.

Thanks again!
